### PR TITLE
Cyborg Reset via Pulse

### PR DESCRIPTION
**Pull request by MarkusLarsson421**
## About The Pull Request
Allows Roboticists to reset Cyborgs via a pulse, mostly using a remote signaler. An even more complicated setup using a proximity sensor or voice analyzer could probably be made to allow them to reset themselves too.

## Why It's Good For The Game
Good for Cyborgs who often wants to swap module but there isn't a dedicated Roboticist or said Roboticist is busy. A simple call over common could be enough for them or someone else to reset them remotely.

## Testing Photographs and Procedure
<details>
<summary>Screenshots&Videos</summary>
The AI shell dropping the BORIS module is unrelated to this PR and isn't the focus on it.

https://github.com/user-attachments/assets/593b6ec8-5699-412e-91ea-683e70a3d2ff

https://github.com/user-attachments/assets/0ccd3bda-3d3c-4bf3-8b63-57025c1191f8

</details>

## Changelog
:cl:
balance: Allows Cyborgs to have their selected module reset via a pulse.
/:cl:

### CODE REVIEW

**Comment by PowerfulBacon** [comment]
This is probably kinda annoying if you are trying to just check the wires since it will reset the borg if you are checking for the law reset wire

**Comment by PowerfulBacon** [comment]
Perhaps have a reset safety wire that needs to be toggled first, or something like that. As long as its obvious why the transformations are failing

**Comment by MarkusLarsson421** [comment]
> Perhaps have a reset safety wire that needs to be toggled first, or something like that. As long as its obvious why the transformations are failing

Well, the reset wire is already highlighted with a star. So if you hit it, it is a certified skill issue, is it not?

**Comment by PowerfulBacon** [comment]
Oh yea, there is. That's unclear though, can you change it to just be clearly marked as the module reset wire?

**Review by Rukofamicom — APPROVED** [review]
Major QoL for cyborgs that swap modules like clothes.

**Comment by Rukofamicom** [comment]
>  can you change it to just be clearly marked as the module reset wire?

This feels like a change that's out of scope ngl. 

Cyborg wire is already marked, roboticists already know or learn it's the one with the star. This PR is just making it so the wire works on pulse as well as cut, instead of only when cut so that a signaler can be installed and used remotely for resets.

**Comment by PowerfulBacon** [comment]
This PR changes the behaviour of pulsing making it much easier to accidentally pulse the reset wire, which is only unclearly marked. Changing the text is a single line of text inside the /datum/wires/cyborg

**Comment by MarkusLarsson421** [comment]
I mean... that's a skill issue. Most people who are or work with Cyborgs know of this and may learn it through the Wiki or word of mouth. Do you still think it has to be made clearer? I've met my fair share of new Roboticists spamming all wires, but we all learn sooner or later.

**Comment by PowerfulBacon** [comment]
No in game content should require the wiki to learn

File: code/datums/wires/robot.dm [hunk 1]
```diff
@@ -23,7 +23,7 @@
 	status += "The intelligence link display shows [R.connected_ai ? R.connected_ai.name : "NULL"]."
 	status += "The camera light is [!isnull(R.builtInCamera) && R.builtInCamera.status ? "on" : "off"]."
 	status += "The lockdown indicator is [R.lockcharge ? "on" : "off"]."
-	status += "There is a star symbol above the [get_color_of_wire(WIRE_RESET_MODULE)] wire."
+	status += "Reset wire: [get_color_of_wire(WIRE_RESET_MODULE)] wire."
```
Comment (PowerfulBacon) [inline code/datums/wires/robot.dm]:
```suggestion
	status += "The [get_color_of_wire(WIRE_RESET_MODULE)] wire is marked 'reset'."
```

Comment (MarkusLarsson421) [reply code/datums/wires/robot.dm]: Even better.

**Pull request merged**